src/components/TokenSelectors/NFTViewer.tsx

```diff
@@ -126,6 +128,18 @@ const LogoIcon = ({ chainId }: { chainId: ChainId }) =>
       src={fantomIcon}
       alt="Fantom"
     />
+  ) : chainId === CHAIN_ID_APTOS ? (
```

Comment:
Later here in the `LogoIcon` component we can use an `Object / Hashmap` and replace the `ternary` chaining.